ui/pages/asset/components/native-asset.tsx

```diff
@@ -29,10 +29,14 @@ const NativeAsset = () => {
   const image = useSelector(getNativeCurrencyImage);
   const showFiat = useSelector(getShouldShowFiat);
   const currentCurrency = useSelector(getCurrentCurrency);
-  const { chainId, ticker, type } = useSelector(getProviderConfig);
+  const { chainId, ticker, type } = useSelector(getProviderConfig) ?? {};
   const { address } = useSelector(getSelectedInternalAccount);
   const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
 
+  if (!chainId) {
+    return null;
+  }
+
```

Comment:
This breaks the rules of hooks... We should not add early returns before any hooks.
See [this](https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level) and details on why React forbids it you can see [this comment from Dan Abramov](https://github.com/facebook/react/issues/15792#issuecomment-497952680)

Reply:
Right.  This branch should not be possible.  You're always on some chain, and `getProviderConfig` should always return one.  Was just trying to satisfy typescript, but I'll do it another way.